stories/core/Breadcrumbs.stories.tsx

```diff
@@ -0,0 +1,87 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { Story, Meta } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import React from 'react';
+import { Breadcrumbs, BreadcrumbsProps } from '../../src/core';
+import { BreadcrumbItem } from '../../src/core/Breadcrumbs';
+import { SvgChevronRightDouble } from '@itwin/itwinui-icons-react';
+
+export default {
+  component: Breadcrumbs,
+  subcomponents: { BreadcrumbItem },
+  title: 'Core/Breadcrumbs',
+  argTypes: {
+    className: { control: { disable: true } },
+    style: { control: { disable: true } },
+    id: { control: { disable: true } },
+    children: { control: { disable: true } },
+  },
+} as Meta<BreadcrumbsProps>;
+
```

Comment:
Maybe add story with dropdown menu and folder icon, just to show how to achieve this and what icon to use.

Reply:
Can we release without this story?

Reply:
Is it really that much work? Just show all items and that's all.

Reply:
Yeah I can do that now. I was hoping we'd have released and demoed today

Reply:
Added. See gif in description.

The code is not very nice and does not handle all cases but it's something 🤷‍♀️

Reply:
I'm using simple dropdown menu items for now.
![image](https://user-images.githubusercontent.com/9084735/127185203-38b5d821-c120-4516-809f-baa873f5eead.png)

@FlyersPh9 Please tell me this atrocity is an outdated standard:
![image](https://user-images.githubusercontent.com/9084735/127185146-bc64c43b-1d3f-48ff-8eeb-f7b0a5b80b45.png)

Reply:
That's been removed from our local version of documentation, it can be ignored. New image below:
![anatomy-breadcrumbs-select](https://user-images.githubusercontent.com/849817/127196299-25802e5f-baeb-4d2d-a60a-fd356800d294.png)